summary-visualiser/templates/scenarios/dht_sync_lag.html.tmpl

```diff
@@ -76,63 +64,23 @@ and there's no function equivalent to `range` like there is for `template`/`tmpl
 
 <div class="scenario-metrics">
     {{ with .scenario_metrics }}
-        {{ with (index .create_rate.rates 0).summary_rate }}
```

Comment:
This is actually a bug fix, and it felt dirty sneaking this into the templating refactoring work in this PR. I'd observed that `.create_rates` in DHT sync lag only ever had one agent, so I cut a corner and just selected the first item in the array. Fixing this to use the new reusable templates is more robust against future scenarios where you might see multiple agents.